install_config/install/advanced_install.adoc

```diff
@@ -125,6 +125,13 @@ link:../../install_config/install/rpm_vs_containerized.html[RPM vs
 Containerized] for more details.
 ifdef::openshift-enterprise[]
 Containerized installations are supported starting in OSE 3.1.1.
+
```

Comment:
The other variables in this table include descriptions of their possible configuration values. For example, the hostname and IP address variables are obviously configured by hostname or IP, and the containerzied variable is described as a *true* or *false* type variable. 

What possible configuration values exist for `*docker_log_driver*`? Or for `*docker_log_options*`? The description tells me what it does, but not how to do it.